pom.xml

```diff
@@ -845,6 +845,9 @@
           </plugin>
         </plugins>
       </build>
+      <properties>
+        <completionGoals>incrementals:reincrementalify</completionGoals>
```

Comment:
BTW it is clearer and safer to put this into a `plugin` section for [`release:prepare`](https://maven.apache.org/maven-release/maven-release-plugin/prepare-mojo.html#completionGoals), rather than relying on a user property which could perhaps be interpreted by unrelated mojos.